packages/mwp-api-proxy-plugin/src/util/duotone.js

```diff
@@ -169,13 +169,7 @@ export const apiResponseDuotoneSetter = duotoneUrls => {
 				groups.forEach(setGroupDuotone);
 				break;
 			case 'home':
-				(value.rows || [])
-					.map(({ items }) => items)
-					.forEach(items =>
-						items
-							.filter(({ type }) => type === 'group')
-							.forEach(({ group }) => setGroupDuotone(group))
-					);
+				(value.most_popular || []).map(event => setGroupDuotone(event.group || {}));
```

Comment:
nitpick, but semantically this should be `forEach` since you're just relying on side effects, not using the new array

Reply:
Thanks, will update!